src/libraries/Microsoft.Extensions.HostFactoryResolver/Directory.Build.props

```diff
@@ -2,5 +2,6 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <PackageDescription>Internal package for sharing Microsoft.Extensions.Hosting.HostFactoryResolver type.</PackageDescription>
+    <IsAspNetCoreApp>true</IsAspNetCoreApp>
```

Comment:
I don't see this library in aspnetcore's list. @dougbu is this one needed?

Reply:
Right, this isn't a library. aspnetcore uses the Microsoft.Extensions.HostFactoryResolver.**Sources** package but (of course) doesn't ship an assembly with this name.

Reply:
@Anipik that's that package, so you want to keep `<IsAspNetCoreApp>` as that's the source package that aspnetcore needs.